packages/studio-be/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "scripts": {
     "start": "cd ./out && cross-env NODE_PATH=./ cross-env BP_MODULES_PATH=B:/bp18/modules node index.js",
-    "build": "yarn clean && yarn tsc",
```

Comment:
It was useful when the structure changed at the beginning, but now it's more a pain in the ass since it will always fail if you already have a botpress instance running with that studio path (then you have to kill the server, run the command again, then start the server